src/structures/MessageMentions.js

```diff
@@ -131,6 +132,10 @@ class MessageMentions {
     const id = data.id || data;
     return this.users.has(id) || this.channels.has(id) || this.roles.has(id);
   }
+
+  toJSON() {
+    return Util.flatten(this, ['members', 'channels']);
```

Comment:
things like arrays of objects should be flattened to their IDs

Reply:
Collections are automatically flattened to an array of their keys.